google-cloud-securitycenter/src/main/java/com/google/cloud/securitycenter/v1/stub/SecurityCenterStubSettings.java

```diff
@@ -704,9 +704,9 @@ public String extractNextToken(ListSourcesResponse payload) {
               .setInitialRetryDelay(Duration.ofMillis(100L))
               .setRetryDelayMultiplier(1.3)
               .setMaxRetryDelay(Duration.ofMillis(60000L))
-              .setInitialRpcTimeout(Duration.ofMillis(480000L))
+              .setInitialRpcTimeout(Duration.ofMillis(20000L))
```

Comment:
The team had explicitly asked about this timeout configuration last week - it's supposed to be 480,000 ms

Reply:
Ack. Updating gapic configs right now and will regenerate once that's public.